x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/service/ServiceAccount.java

```diff
@@ -5,7 +5,7 @@
  * 2.0.
```

Comment:
These had to move since they're now part of the `SecurityExtension` interface through `ServiceAccountTokenStore`.